scripts/zones/Kazham/npcs/Magriffon.lua

```diff
@@ -35,9 +61,15 @@ end;
 -----------------------------------
 
 function onEventFinish(player,csid,option)
---printf("CSID: %u",csid);
---printf("RESULT: %u",option);
+    if(csid == 0x0090 and option == 1)  then    -- Gullible's Travels: First CS
+        player:addQuest(OUTLANDS, GULLIBLES_TRAVELS);
+    elseif(csid == 0x0092) then    -- Gullible's Travels: Final CS
+        player:tradeComplete();
+        player:addGil(tradedGil - player:getVar("MAGRIFFON_GIL_REQUEST"));    -- Returns extra gil paid
+        tradedGil = 0;

```

Comment:
instead of completing the trade and refunding extra gil, if you use confirmTrade without confirming any items, it won't take any items away.  then, you can just delete the gil manually (since you know how much gil to delete in the players variable), and you won't have to try and save a variable across functions